webui/react/src/pages/Cluster/ClusterQueuedChart.tsx

```diff
@@ -38,13 +36,14 @@ const ClusterQueuedChart: React.FC<Props> = ({ poolStats }: Props) => {
       <Section
         bodyBorder
         options={
-          <Radio.Group
-            className={css.filter}
+          <RadioGroup
+            options={[
+              { id: 7, label: '7 days' },
+              { id: 30, label: '30 days' },
+            ]}
             value={viewDays}
-            onChange={(e) => setViewDays(e.target.value)}>
-            <Radio.Button value={7}>7 days</Radio.Button>
-            <Radio.Button value={30}>30 days</Radio.Button>
-          </Radio.Group>
+            onChange={(value) => setViewDays(value as number)}
```

Comment:
nit: similarly, making RadioGroup generic would allow for this assertion to be removed